src/popup/main.tsx

```diff
@@ -13,17 +152,127 @@ const Popup = () => {
+                  type="text"
+                  placeholder="https://codecov.example.com"
+                  className={clsx(
+                    "input input-bordered w-full",
+                    (isUrlError || isTokenError) && "border-2 border-red-500"
+                  )}
+                  value={codecovUrl}
+                  onChange={handleTextChange(setCodecovUrl)}
+                />
+                {isUrlError && (
+                  <label className="label">
+                    <span className="label-text-alt text-red-500">
+                      Request failed
+                    </span>
+                  </label>
+                )}
+              </div>
+              <div>
+                <label className="label">
+                  <span className="label-text">Codecov API Token</span>
+                </label>
+                <input
+                  type="text"
+                  placeholder="1a2bc3de-f45g-6hi7-8j90-12k3l45mn678"
+                  className={clsx(
+                    "input input-bordered w-full",
+                    isTokenError && "border-2 border-red-500"
+                  )}
+                  value={codecovApiToken}
+                  onChange={handleTextChange(setCodecovApiToken)}
+                />
+                {isTokenError && (
+                  <label className="label">
+                    <span className="label-text-alt text-red-500">
+                      Authentication failed
+                    </span>
+                  </label>
+                )}
+              </div>
+              <div>
+                <label className="label">
+                  <span className="label-text">GitHub URL</span>
+                </label>
+                <input
+                  type="text"
+                  placeholder="https://github.example.com"
+                  className={clsx(
+                    "input input-bordered w-full",
+                    isTabError && "border-2 border-red-500"
+                  )}
+                  value={githubUrl}
+                  onChange={handleTextChange(setGitHubUrl)}
+                />
+                <label className="label">
+                  <span
+                    className={clsx(
+                      "label-text-alt",
+                      isTabError && "text-red-500"
+                    )}
+                  >
+                    This URL must be loaded in the active tab
+                  </span>
+                </label>
+              </div>
+            </div>
+          )}
+        </div>
+        {isUnregisterTabError && (
+          <div>
+            <label className="label">
+              <span className="label-text text-red-500">
+                The self hosted GitHub instance must be loaded in the active tab
+              </span>
+            </label>
+          </div>
+        )}
+        <div className="divider" />
+        <div>
+          The Codecov Browser Extension is currently in Beta. <br />
```

Comment:
```suggestion

```

Reply:
Given that this has been released for a few weeks and we have handled a majority of issues/feedback, I feel like we can remove the Beta designation.